concreep-refilled/control.lua

```diff
@@ -64,14 +64,8 @@ function init()
 	global.reinit_current_surface_roboport_index = 1
 	global.reinit_level_running = 0
 
-	for each, surface in pairs(game.surfaces) do
-		local roboports = surface.find_entities_filtered{type="roboport"}
-		for index, port in pairs(roboports) do
-			if is_valid_roboport(port) then
-				add_creeper_for_roboport(port)
-			end
-		end
-	end
+	script.on_nth_tick(nil) --unregister all nth tick handlers, so we can update the interval n
+	script.on_nth_tick(run every n updates, creepers_update)

```

Comment:
missing settings.global ?

Reply:
whops - ye must have not used replace all in this instance.

Reply:
fixed